lib/msal-common/src/error/AuthError.ts

```diff
@@ -52,6 +53,11 @@ export class AuthError extends Error {
      */
     correlationId: string;
 
+    /**
+     * Default NativeAuthError from MsalNodeRuntime when broker is enabled
+     */
+    msalNodeRuntimeError?: NativeAuthError;
```

Comment:
Let's make sure this is general enough to encompass our browser scenarios as well
```suggestion
    platformBrokerError?: PlatformBrokerError;
```